src/org/jitsi/impl/neomedia/MediaStreamImpl.java

```diff
@@ -3653,6 +3653,21 @@ public void injectPacket(RawPacket pkt, boolean data, TransformEngine after)
      */
     public int getTemporalID(byte[] buf, int off, int len)
     {
+        if (frameMarkingsExtensionId != -1)
+        {
+            RawPacket pkt = new RawPacket(buf, off, len);
```

Comment:
Me too, I started with the patch, but if I created the RawPacket outside of the function and pass it to the inner layer, everything stop working. I was getting crazy, so I decided to keep it for later.. :)